addons/binding/org.openhab.binding.silvercrestwifisocket/src/main/java/org/openhab/binding/silvercrestwifisocket/runnable/SilvercrestWifiSocketUpdateReceiverRunnable.java

```diff
@@ -0,0 +1,92 @@
+package org.openhab.binding.silvercrestwifisocket.runnable;
```

Comment:
you should move ALL classes that are internal to your binding into org.openhab.binding.silvercrestwifisocket.internal...